src/Modules/OrchardCore.Commerce/Views/TextProductAttributeField.cshtml

```diff
@@ -6,6 +6,12 @@
     var settings = attr?.Settings as TextProductAttributeFieldSettings;
     var id = $"{Model.IdPrefix}-attribute-{index}";
     var optionIndex = 0;
+
+    var sku = (string)Model.Sku;
```

Comment:
I would use `Model.Sku as string` as this will return null if the cast fails.
Ir it is intentional to have an exception if it fails then I am okay with this.

Reply:
Realistically, SKU should never be empty on ProductPart, but I'll use the `as string` approach for consistency.